R/ff_load_config.R

```diff
@@ -0,0 +1,17 @@
+ff_load_config <- function(config_path = "config.json") {
```

Comment:
is it necessary to have ff_ in the beginning of the file name?

Reply:
it is used to avoid any further conflict. 
since i am not sure why config file cannot be used in testthat so i will not take any risk of conflicts by adding the ff_ prefix.

Reply:
initialize_config or config_init ?

Reply:
OK, I changed ff_load_config to config_load. I tested it and it's working.